charts/GPP-stack/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v2
+name: gpp-stack
+description: An umbrella chart for the GPP stack
+
+type: application
+
+# Chart version
+version: 0.1.0
+
+appVersion: "0.1.0"
+
+dependencies:
+  - name: openzaak
+    version: 1.3.0
+    repository: https://maykinmedia.github.io/charts/
+    condition: openzaak.enabled
+
+  - name: gpp-app
+    version: latest
+    repository: https://GPP-Woo.github.io/charts
+    condition: gpp-app.enabled
+
+  - name: gpp-publicatiebank
+    version: main
```

Comment:
if this refers to the image tag, then it should be `latest` for publicatiebank and `main` for gpp-app I think

Reply:
urrrgh yes, you are right! I keep mixing up the two names!! :see_no_evil: